web/serve_locally.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+# This serves the current directory over HTTP. We need something more than
```

Comment:
The CORS stuff is because the .html page grabs Leaflet stuff from unpkg. The browser has special rules to stop one website from downloading stuff from other domains. I don't understand all of it, but this is the simplest workaround I found so far for running things locally. I'm sure there's a better way with the npm javascript tool. Will keep iterating.